runtime/ops/os/sys_info.rs

```diff
@@ -0,0 +1,281 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+
+type LoadAvg = (f64, f64, f64);
+const DEFAULT_LOADAVG: LoadAvg = (0.0, 0.0, 0.0);
+
+pub fn loadavg() -> LoadAvg {
+  #[cfg(target_os = "linux")]
+  {
+    fn read_loadavg() -> Result<String, std::io::Error> {
+      let mut s = String::new();
+      File::open("/proc/loadavg")?.read_to_string(&mut s)?;
```

Comment:
in general, using procfs is quite slow. there are alternatives we can use on modern linux for most of these places we read procfs. this also has benefit of not being dependent on procfs being mounted.

i raised an issue here on same theme: https://github.com/denoland/deploy/issues/3813

for loadavg, there is a sysinfo() syscall which gives us load average as well as all the memory stats. are there reasons we prefer to use procfs and/or cannot use sysinfo(), getrusage(), sysconf() etc. on linux?

https://man7.org/linux/man-pages/man2/sysinfo.2.html

Reply:
i did a little benchmark and pulling uptime etc. from sysyinfo syscall is ~2x doing an lseek/read on procfs for uptime alone (1.8m ops/sec v 800k). and it has the memory info we need in the same struct.